### DISCUSSION TITLE
Webdav performance is extremely slow

### DISCUSSION THREAD

**Discussion by Joly0** [nextcloud/all-in-one#4681] [Questions] [answered]
### Steps to reproduce
1. Add webdav share to windows explorer
2. Upload files

### Expected behavior 
All files should be uploaded normally and with decent speed (my setup is local, so over 2.5G ethernet)
### Actual behavior 
Files take forever to upload. Large files upload fast at the start, but stop at 99% of upload and then are stuck there for minutes.
Small files are uploading, but with very bad speed (i am talking 10KB/s)

### Host OS 
Unraid

#### Nextcloud AIO version 
Nextcloud AIO v8.2.1
#### Other valuable info 
I have tried using the nextcloud application on windows and that works. Upload speeds look good so far.

**Comment by szaimen** [comment, marked as answer]
Hi, do you have clamav enabled? If so, disabling it might speed up the transfer. Apart from that, see https://github.com/nextcloud/all-in-one/discussions/1335
